src/Components/ResourcePicker/ResourcePicker.tsx

```diff
@@ -32,6 +32,7 @@ import {
     AzureResourceDisplayFields,
     AzureResourceTypes,
     IAzureResource,
+    IComponentError,
```

Comment:
![image](https://user-images.githubusercontent.com/57726991/186237258-b2b6dbe3-3fb6-4465-8084-e90ac5f937a3.png)

Reply:
had commented on this on the original PR